lib/api/api_config.rb

```diff
@@ -3,6 +3,7 @@
 module Api
   ApiConfig = ::Config::Options.new.tap do |o|
     o.add_source!(ManageIQ::Api::Engine.root.join("config/api.yml").to_s)
+    Vmdb::Plugins.each { |plugin| o.add_source!(plugin.root.join('config/api.yml').to_s) }
```

Comment:
Do we want to put a rescue here in case there's a provider that introduces some malformed yaml?

Reply:
It only fails on the `.load!` no on the `.add_source!` if there is malformed YAML and at that point I'm not sure how we would handle it.
Blowing up quickly has its benefits :D